src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -595,22 +870,18 @@ end
 		itemId = "Red Paintball Gun"
 	})
 	```
-	
+
 	@param data PurchaseEvent
-	@return { [any]: any }
+	@return Promise
```

Comment:
```suggestion
	@return Promise<T>
```
Unless the type is already defined, you need to define custom return types with Moonwave by denoting them with a after the type (Eg. `Promise<T>`)